Dangerfile

```diff
@@ -0,0 +1,27 @@
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+if github.pr_title.include? "[WIP]"
+  warn "PR is classed as Work in Progress"
+end
+
+# Warning to encourage a PR description
+if github.pr_body.length == 0
+    warn "Please provide a summary in the PR description to make it easier to review"
+end
+
+# Ensure that labels have been used on the PR
+if github.pr_labels.empty?
+    failure "Please add labels to this PR"
+end
+
+# Make danger out jira ticket link
+jira.check(
+  key: ["SAAS"],
```

Comment:
what does the key do here ?

Reply:
The way it'll work is, you put the key in PR title or body and danger will produce the link in the comment.  For instance I put [SAAS-11757] in the title and danger commented the ticket link. 
And if we go further and integrate jira with github, then jira will use these keys from PR titles and add the PR/branch to the ticket automatically.